skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/model/Identification.java

```diff
@@ -29,41 +35,63 @@ public String getBusinessKey() {
         return businessKey;
     }
 
+    public long getStartTime() {
+        if (startTime == 0){
+            return System.currentTimeMillis();
```

Comment:
This is an identification, only be used in create span. Why need `getStartTime`?
And even need this, why not assign `this.startTime` = 'return-value'?